libsmtutil/Z3Interface.cpp

```diff
@@ -21,12 +21,23 @@
 #include <libsolutil/CommonData.h>
 #include <libsolutil/CommonIO.h>
 
-#include <z3_api.h>
+#ifdef HAVE_Z3_DLOPEN
+#include <libsmtutil/Z3Loader.h>
+#endif
 
 using namespace std;
 using namespace solidity::smtutil;
 using namespace solidity::util;
 
+bool Z3Interface::available()
+{
+#ifdef HAVE_Z3_DLOPEN
+	return Z3Loader::get().available();
+#else
```

Comment:
```suggestion
#endif
```

Reply:
I don't get that comment.

Reply:
if `HAVE_Z3_DLOPEN` is true it will return early, so no need for an else branch, can just always return true

Reply:
Then we have dead/unreachable-code, for which, if we don't have one yet, @axic will probably soon introduce a compiler warning breaking the build :-D...

Reply:
Haha ok, fine as is then.